packages/athena/libs/other_apis_lib.js

```diff
@@ -345,6 +354,11 @@ module.exports = function (logger, ev, t) {
 
 				// couchdb setting edits
 				if (req.body.default_user_password) {
+					const result = owasp.test(req.body.default_user_password);
+					if (result && Array.isArray(result.errors) && result.errors.length > 0) {
+						logger.error('[default pass] not a viable password. rule failures');
```

Comment:
It would be nice to include the actual errors in the message, something like this:

```suggestion
						logger.error(`[default pass] Password rule failures: ${result.errors.join(', ')}`);
```